docs/guides/introduction/about.md

```diff
@@ -1 +1,23 @@
-# About Daffodil
+# Introduction
+
+## What is Daffodil?
+
+Daffodil is a Typescript framework for building headless ecommerce stores. It is built on top of Angular and provides a suite of composable packages that allow you to build complicated ecommerce stores.
+
+These packages, in turn, provide tools and architectures that allow you to quickly build storefronts that are:
+
+1. Performant
+2. Reliable
+3. Interoperable
+4. Maintainable
+5. Accessible
+
+## The Framework
+
+Daffodil is built with flexibility and scalability in mind. Here are a few features that we believe make Daffodil special:
+
+- **Drivers** [Drivers](/guides/drivers) are Daffodil's lifeblood. The drivers enable you to easily integrate with a variety of eCommerce backends, payment systems, and other services, allowing you to extend functionality without disrupting the core framework.
```

Comment:
This link exists, but it should be `/docs/guides/drivers.md`

Reply:
Fixed.